sdk/search/azure-search-documents/CHANGELOG.md

```diff
@@ -2,13 +2,9 @@
 
 ## 11.8.0-beta.4 (Unreleased)
```

Comment:
Don't forget to update the date and the references to the version in READMEs

Reply:
@alzimmermsft I'm going to make a separate PR for the actual release. I try to keep release PRs separate from any PRs that update the SDKs to avoid any mishaps during releases.